### PR TITLE
PROGRAMMES-6707 - Count method

### PR DESCRIPTION
**Pull request by cisco87**
Needed by https://github.com/bbc/programmes-frontend/pull/810

### CODE REVIEW

File: src/Service/ProgrammesAggregationService.php [hunk 1]
```diff
@@ -42,6 +42,24 @@ public function findStreamableDescendantClips(
         return $this->findStreamableDescendantsByType($programme, 'Clip', $limit, $page, $ttl, $nullTtl);
     }
 
+    public function countStreamableDescendantClips(
+        Programme $programme,
+        $ttl = CacheInterface::NORMAL,
+        $nullTtl = CacheInterface::NORMAL
+    ) : int {
+        $key = $this->cache->keyHelper(__CLASS__, __FUNCTION__, $programme->getPid(), 'ClipsCount', null, null, $ttl);
```
Comment (BjornTwachtmann) [inline src/Service/ProgrammesAggregationService.php]: You don't need the two null parameters here, or that clipscount string

Comment (cisco87) [reply src/Service/ProgrammesAggregationService.php]: Sorry about the 2 nulls, but I tought the ClipsCount was a sort of prefix to make the cache key unique?

Comment (cisco87) [reply src/Service/ProgrammesAggregationService.php]: ok I understand now, it's the __FUNCTION__ that will help generate it, got it!

File: src/Service/ProgrammesAggregationService.php [hunk 2]
```diff
@@ -42,6 +42,24 @@ public function findStreamableDescendantClips(
         return $this->findStreamableDescendantsByType($programme, 'Clip', $limit, $page, $ttl, $nullTtl);
     }
 
+    public function countStreamableDescendantClips(
+        Programme $programme,
+        $ttl = CacheInterface::NORMAL,
+        $nullTtl = CacheInterface::NORMAL
+    ) : int {
+        $key = $this->cache->keyHelper(__CLASS__, __FUNCTION__, $programme->getPid());
```
Comment (BjornTwachtmann) [inline src/Service/ProgrammesAggregationService.php]: You do, however, need the TTL in there

**Pull request merged**